src/schema/nmdc.yaml

```diff
@@ -1869,11 +1872,13 @@ classes:
         comments:
           - pattern should allow typecode for Biosample and ProcessedSample
       part_of:
-        range: Study
-        pattern: "^nmdc:sty-[0-9][a-z]{0,6}[0-9]-[A-Za-z0-9]{1,}(\\.[A-Za-z0-9]{1,})*(_[A-Za-z0-9_\\.-]+)?$" # better applied as a structured_pattern, but even that might get out of sync from the asserted Study structured_pattern
+        range: DataGeneration
+        pattern: "^nmdc:datgen-[0-9][a-z]{0,6}[0-9]-[A-Za-z0-9]{1,}(\\.[A-Za-z0-9]{1,})*(_[A-Za-z0-9_\\.-]+)?$" # better applied as a structured_pattern, but even that might get out of sync from the asserted Study structured_pattern
```

Comment:
I think we should remove this in favor of enforcing the pattern in the Classes and then specifying the range as you have done.